src/current/cockroachcloud/tools-page.md

```diff
@@ -64,8 +71,8 @@ To monitor the health of metrics export, you can [create a custom Monitor](#moni
 
 Open your Datadog [Dashboard List](https://docs.datadoghq.com/dashboards/#dashboard-list). There are two sample dashboards that present CockroachDB metrics:
 
-- `CockroachDB {{ site.data.products.cloud }} {{ site.data.products.standard }}`
-- `CockroachDB {{ site.data.products.cloud }} {{ site.data.products.advanced }}`
+- `CockroachDB {{ site.data.products.cloud }} {{ site.data.products.serverless }} (Limited Preview)`
```

Comment:
Since we're linking to the Datadog docs that will be using the old terminology until they deploy their doc updates, it would be great to update the terminology on our side use the new Standard and Advanced terms but append "(formerly Serverless)" and "(formerly Dedicated") so there's continuity when going to the Datadog docs.